roles/webserver/templates/unicorn_environment.j2

```diff
@@ -1,2 +1,5 @@
 RAILS_ENV={{ rails_env }}
+{% if skylight_authentication is defined %}
```

Comment:
let's just remove it. I wanted to clean this up at some point because we abandoned skylight since https://github.com/openfoodfoundation/openfoodnetwork/commit/ab8c7bad7696b2684e8725e4902b509815aa9b48